vm/rust/Cargo.toml

```diff
@@ -7,12 +7,14 @@ edition = "2021"
 
 [dependencies]
 serde = "1.0.160"
-serde_json = "1.0.96"
-blockifier = {git = "https://github.com/starkware-libs/blockifier", rev = "d571467"}
+serde_json = { version = "1.0.96", features = ["raw_value"] }
+blockifier = {git = "https://github.com/starkware-libs/blockifier", rev = "v0.1.0-rc0"}
 starknet_api = { git = "https://github.com/starkware-libs/starknet-api", rev = "a4c78ff" }
 cairo-vm = "0.6"
-cairo-lang-casm = "2.0.0-rc4"
-cairo-lang-starknet = "2.0.0-rc4"
+cairo-lang-casm = "2.0.0"
+cairo-lang-starknet = "2.0.0"
+
+starknet = { rev = "ada5db0", git = "https://github.com/xJonathanLEI/starknet-rs" }
```

Comment:
Should we use release for this as well instead of a commit?

Reply:
Yeap, will do